building-a-preview/how-previews-work/index.md

```diff
@@ -0,0 +1,213 @@
+required to generate a working Preview.
+
+In addition to speeding up your Preview builds, Tugboat saves disk space by
+storing only a binary difference between the Base Preview and Previews built
+from that Base Preview. A new Preview only uses whatever space it needs that
+differs from the Base Preview. Often, this means a Base Preview might use 2-3GB
+of space, and a Preview built from it might only use 100-200MB. This is a great
+way to keep a Tugboat Project under your
+[billing tier's storage limit](../../tugboat-billing/index.md#how-does-tugboat-pricing-work),
+even when you're building multiple Previews.
+
+> #### Hint:: Base Previews and Preview Build Stages
+>
+> When you set a Base Preview, new Previews you build - including Previews that
+> are built automatically from pull requests - use the Base Preview as a
+> starting point, and build only from the `build` stage. This means that if
+> you're making changes that would be processed during `init` or `update`
+> stages, or changing a Docker image, you'll need to either
+> [rebuild the Base Preview](../work-with-base-previews/index.md#change-a-base-preview),
+> or
+> [build the Preview from scratch without the Base Preview](../work-with-base-previews/index.md#build-a-preview-with-no-base-preview).
+> For more info, see:
+> [Building and Rebuilding Previews when you're using a Base Preview](../work-with-base-previews/index.md#building-and-rebuilding-previews-when-youre-using-a-base-preview).
+
+## Preview size: explained
+
+What Tugboat calls "Preview size" is actually the size of the container in which
+the Preview lives. When a Preview is done building, Tugboat takes a snapshot of
+the container at that moment in time, and that's what you're seeing in your
+Tugboat Dashboard.
+
+When you [set up Services](../../setting-up-services/index.md) in your Preview,
+Tugboat pulls those
+[Service images](../../setting-up-services/how-to-set-up-services/index.md#specify-a-service-image)
+into the Preview container. Each of these Service images contributes to the
+total size of the Preview when it is fully built.
+
+For example, say I'm building a Preview that uses Tugboat's Service images for
+`apache`, `mysql` and `redis`; those Service images are 154MB, 226MB, and 147MB
+at the time of this writing. That's over 500MB for these Services; by the time
+you add a database file, the container's operating system and other assets,
+you'll likely be looking at 800MB to 1GB for a Preview that's only pulling in
+100KB of code from your linked git repo.
+
+This is why working from a [Base Preview](#how-base-previews-work) is so helpful
+in reducing Preview size; all of those assets are contained in the Base Preview.
+When you build a new Preview from the Base Preview, the new Preview only
+contains what's different in the PR, Branch or Tag you're building. In my
+example above, a new Preview built from the Base Preview was only 20KB.
+
+## Preview status
```

Comment:
I always wondered if this list was actually useful. We originally drew it up because it takes up space (if we're being honest with ourselves). I don't have particularly strong opinions on whether to keep or delete it, but it definitely feels like "fluff" to me.

Reply:
It might be useful if a user actually has a question... and if no users need it, it's at the bottom of a page anyway, so it's not necessarily in a high-priority place. I don't mind it staying, so in my mind it's really a question of whether we want to remove it from a maintenance standpoint; i.e. if any of this changes, do we want to keep and update this section? Or should we remove this section so we don't have to maintain it?

Reply:
We can leave it. We always have the option to delete it later if we really want to.